src/views/settings/Storage/CommandStorage.vue

```diff
@@ -74,10 +78,15 @@ export default {
           actions: {
             formatterArgs: {
               canUpdate: function({ row }) {
-                return (row.name !== 'default' && row.name !== 'null' && vm.$hasPerm('terminal.change_commandstorage'))
+                return (row.name !== 'default' && row.name !== 'null' &&
+                  vm.$hasPerm('terminal.change_commandstorage'))
               },
               canDelete: function({ row }) {
-                return (row.name !== 'default' && row.name !== 'null' && vm.$hasPerm('terminal.delete_commandstorage'))
+                return (row.name !== 'default' && row.name !== 'null' &&
+                  vm.$hasPerm('terminal.delete_commandstorage'))
+              },
+              default: {
+                width: '130px'
               },
               hasClone: false,
               extraActions: [
```

Comment:
There are several issues to consider:

  * `vm` is not defined before its use

This can lead to an error at runtime, since Vue does not know about it until you initialize your component. This might be because of a missing import statement or a typo.

I recommend:

```
import {ref} from 'vue'
// ...
const vm = ref(null)
```

However, there may be some other problems depending on where this file belongs and how components are being imported. Please review and update the code accordingly.